tiatoolbox/visualization/tileserver_api.yml

```diff
@@ -197,10 +197,10 @@ paths:
               properties:
                 bounds:
                   type: list
-                  description: a list of 4 numbers, [xmin, ymin, xmax, ymax] describing the bounds of the query
+                  description: a json encoded list of 4 numbers, [xmin, ymin, xmax, ymax] describing the bounds of the query
                 where:
                   type: string
-                  description: a valid AnnotationStore filter string
+                  description: a valid AnnotationStore filter string, json encoded
```

Comment:
a -> A

Reply:
I've corrected the capitalization in the instances you pointed out and a few other ones too